### PR TITLE
Fixes #2332 : Translate App Strings to Spanish by merlox

### LINKED ISSUE #2332: Translate App Strings to Spanish
### User Story

As a Spanish language speaker, I want to be able to interact with Status in my native language.

### Description

*Type*: Feature

*Summary*: We need someone fluent (and eloquent) in Spanish to translate all the latest strings in Status from English. We would like to invite our community members to take on these tasks in order
a) To become a bit more acquainted with the Status repos and perhaps wander further into the codebase and set of issues we need help with, earning yourself Open Bounties all the way down this rabbit hole, and
b) To make sure that the product we deliver has been built by the same community that uses it, in a virtuous feedback loop of increasing understanding and value-generation.

#### Expected behavior
The Spanish translation ought to be as up-to-date as the English one.

#### Actual behavior
We have not yet updated it, preferring to leave this open as an opportunity for our community to get involved again.

### Solution

*Summary*: 

- The file [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/es.cljs) needs to be updated to match the new English one [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/en.cljs).
- How you translate is a matter of your preference. There were significant updates to the English file for the 0.9.12 release, with lots of strings both add and REMOVED. We need you to make sure that the files are exact mirrors of each other, except in different languages.
- It might be easier to simply take the new English file and translate from scratch rather than going through the current translated files line by line (though we leave this up to you).

### Additional Information

* Status version: 0.9.12+

There are bounties available for these translations. Please see the QR codes that will be added below after the DevCon 3 announcement.

### PR DESCRIPTION
**Pull request by merlox**
Fixes #2332

This is the translation to spanish. I've changed the first line from:
```
(ns status-im.translations.en)
```
To:
```
(ns status-im.translations.es)
```

### CODE REVIEW

**Comment by adnx2h** [comment]
mainnet and testnet should be translated as well

**Comment by LaloGarza** [comment]
There’s some translations that are not correct, for example line 105, it should be: Nuevo chat grupal.
And line 137 the word Unirse is misspelled.
57bda6f0-ee4a-41c8-804a-18b1e4b02e8c

File: src/status_im/translations/es.cljs [hunk 1]
```diff
@@ -2,50 +2,67 @@
...
-   ;messages
-   :status-sending                        "Enviando"
+   ;;messages
    :status-pending                        "Pendiente"
    :status-sent                           "Enviado"
    :status-seen-by-everyone               "Visto por todos"
    :status-seen                           "Visto"
    :status-delivered                      "Entregado"
-   :status-failed                         "Fallido"
+   :status-failed                         "Fallído"
```
Comment (weisk) [inline src/status_im/translations/es.cljs]: va sin acento

File: src/status_im/translations/es.cljs [hunk 2]
```diff
@@ -54,254 +71,295 @@
...
-   :phew-here-is-your-passphrase          "*Uf* ha sido difícil, aquí está tu contraseña, * escríbela y ¡guárdala en un sitio seguro! * La necesitarás para recuperar tu cuenta."
+   :browsing-title                        "Navegar"
+   :browsing-browse                       "@navega"
+   :browsing-open-in-web-browser          "Abrir en el navegador web"
+   :browsing-cancel                       "Cancelar"
+
+   ;;sign-up
+   :contacts-syncronized                  "Tus contactos se han sincronizado"
+   :confirmation-code                     (str "¡Gracias! Has enviado un mensaje de texto con una confirmación "
+                                               "código. Por favor danos ese código para confirmar tu número de teléfono")
```
Comment (weisk) [inline src/status_im/translations/es.cljs]: "con una confirmacion código" no tiene sentido. Mejor poner "con un código de confirmacion". El salto de línea da igual que lo pongas con unas palabras más o menos de margen.

File: src/status_im/translations/es.cljs [hunk 3]
```diff
@@ -54,254 +71,295 @@
...
+   :here-is-your-signing-phrase           "Aquí está tu frase para firmar. La vas a usar para firmar tus transacciones. *¡Anótala y mantenla a salvo!*"
+   :phone-number-required                 "Toca aquí para validar tu número de teléfono y encontraré a tus amigos."
+   :shake-your-phone                      "¿Has encontrado un error o tienes alguna sugerencia? ¡Simplemente ~agita~ tu teléfono!"
+   :intro-status                          "Habla conmigo para configurar tu cuenta y cambiar tus ajustes."
+   :intro-message1                        "¡Bienvenido a Status!\nToca este mensaje para establecer tu clave y empezar."
    :account-generation-message            "Dame un segundo, ¡tengo que hacer unas cálculos para generar tu cuenta!"
+   :move-to-internal-failure-message      "Tenemos que mover algunos archivos importantes del almacenamiento externo al interno. Para ello necesitamos tu permiso. No usaremos el almacenamiento externo en versiones futuras."
+   :debug-enabled                         "¡El servidor de depuración ha sido activado! Ahora puedes ejecutar *status-dev-cli scan* para encontrar el servidor en tu ordenador desde la misma red."
```
Comment (weisk) [inline src/status_im/translations/es.cljs]: "en la misma red"

File: src/status_im/translations/es.cljs [hunk 4]
```diff
@@ -54,254 +71,295 @@
...
+
+   ;;phone types
+   :phone-e164                            "Internacional 1"
+   :phone-international                   "Internacional 2"
+   :phone-national                        "Nacional"
+   :phone-significant                     "Significante"
```
Comment (weisk) [inline src/status_im/translations/es.cljs]: Significativo

File: src/status_im/translations/es.cljs [hunk 5]
```diff
@@ -54,254 +71,295 @@
...
+   :gas-used                              "Gas usado"
+   :cost-fee                              "Coste/Tarifa"
+   :nonce                                 "Nonce"
+   :confirmations                         "Confirmaciones"
+   :confirmations-helper-text             "Por favor espera al menos a que haya 12 confirmaciones para asegurarte de que tu transacción es procesada con seguridad"
+   :copy-transaction-hash                 "Copiar hash de transacción"
+   :open-on-etherscan                     "Abrir en Etherscan.io"
+
+   ;;webview
+   :web-view-error                        "oops, error"
```
Comment (weisk) [inline src/status_im/translations/es.cljs]: una tontería, pero creo que en español es mejor "ups"

**Review by weisk — CHANGES_REQUESTED** [review]
revisiones

**Comment by yenda** [comment]
@merlox can this PR be closed ? It only removes 2 empty lines in the translation file ?

**Comment by oskarth** [comment]
Closing this PR as it doesn't include any translations. Feel free to open another PR if you feel like this is an error.

**Pull request closed without merging**